### ISSUE TITLE
Need extra scroll padding on right of theme select

### ISSUE THREAD

**Issue by evadecker** [namesakefyi/namesake#635]
![image](https://github.com/user-attachments/assets/6f1f8475-087b-4927-95f1-25d02130d9d1)

It would also be nice to include a tooltip/blurb explaining the colors and meanings from the Gilbert Baker pride flag

Additionally, the scroll region is cutting off focus rings and should have vertical padding increased to prevent that

![image](https://github.com/user-attachments/assets/67d553fb-9861-47f9-a100-9340f14d3201)
